### ISSUE TITLE
Mobile layout

### ISSUE THREAD

**Issue by Xon** [Sidane/xenforo-threadmarks#27]
In portrait mode, on my galaxy nexus, the "got unread" goes away if there are thread marks.

In landscape mode, the threadmark button isn't aligned with the unread button properly

**Comment by sidane** [comment]
Screenshot?

**Comment by Xon** [comment]
![example](http://puu.sh/fvh8f/5b53fdf526.png)

**Comment by sidane** [comment]
Turns out XF hides the unread link on narrow viewports!

See `page_nav.css`:

```
@media (max-width:@maxResponsiveNarrowWidth)
{
  .Responsive .PageNav .unreadLink
  {
    display: none;
  }
}
</xen:if> 
```

The landscape issue might(?) be unavoidable as I think the pagination links are meant to wrap down to the next line if there's not enough space to fit it on one line. Can you verify?

I think it's worse for mods/admins/thread starters as they have the `Thread tools` and `Watch thread` links on the same line as the pagination which reduces the available space.

**Comment by Xon** [comment]
The Thread starter also can have the same Thread Tools options. Playing around with Chrome's emulation stuff, the pagination links do wrap to the next line. But you need to make the screen utterly unusable small.

However, having a phone/tablet where the unread button was there, and then suddenly is not after a feature is unused in the thread is not in my mind desirable behaviour. If they never had the button, it would be a different story.

**Comment by sidane** [comment]
As a thread starter, I'm seeing the following at ~500px width, a reasonable viewport size:

![Pagination wrapping](http://f.cl.ly/items/0V3O281R3P0F08173g2D/pagination-wrapping2.png)

The lack of top margin on the  `Go to first unread` button looks like a styling bug in XF to me.

When logged in as a non-thread starter (so only `Watch Thread` is visible) the styling is setup to ensure the pagination never wraps. Given this is the behaviour the majority of users see, I think we should ensure it stays that way when the threadmarks is there. I'll look into that now.

As pagination wrapping for mods/admins/thread starters is already happening in XF I think we should leave that as is.

I'm not sure what you mean in your second paragraph above. If you're referring to the `Go to first unread` button disappearing on smaller viewports then that's XF's styling which again, I don't think we should change.

**Comment by Xon** [comment]
Sorry,I was being very verbose in saying; 
- If they have an unread button with a given resolution, they should keep having one.

**Comment by sidane** [comment]
So you're saying the XF behaviour is the problem? This isn't an issue we need to address in this add-on?

**Comment by Xon** [comment]
For borderline resolutions, adding a threadmark button will make the unread button go away. I would prefer to avoid that.

**Comment by sidane** [comment]
I'm not sure at what resolutions this happens.

Can you try PR #34 and see if it fixes it?

**Comment by Xon** [comment]
Ok, testing now

**Comment by sidane** [comment]
Fixed by #34.
